src/components/ConnectToSageIntacctButton/index.tsx

```diff
@@ -57,6 +61,13 @@ function ConnectToSageIntacctButton({policyID, shouldDisconnectIntegrationBefore
         <>
             <Button
                 onPress={() => {
+                    if (policy?.type !== CONST.POLICY.TYPE.CORPORATE) {
+                        Navigation.navigate(
```

Comment:
BUG: An additional item is put on navigation stack, so you are taken to the workspace page two times.

Suggestion:  Use `navigation.dismissModal()` before navigating to see if it fixes the issue.


https://github.com/Expensify/App/assets/29673073/41f8cd3e-9dfa-40d9-89ad-e24ecc2c250a

Reply:
Fixed!

Reply:
Also it looks like the Sage image is not rounded here:
![CleanShot 2024-07-16 at 15 09 22@2x](https://github.com/user-attachments/assets/e1c552f6-54df-4273-973a-6be463d498ce)

Reply:
Also, does the bolded $9 need to be in a different color than the rest of the text? I don't know that we had that in our mockups?

Reply:
@shawnborton Handled.

<img width="412" alt="Screenshot 2024-07-17 at 8 28 29 PM" src="https://github.com/user-attachments/assets/553d9396-e35a-488c-8ce3-f42a3bb4cce1">

Reply:
While I have your attention, what is the inner padding of the upgrade card? It should be 20px but the screenshot above makes it seem like it might only be 16px.

Reply:
Yep. I'm updating this to `20px` as part of the PR in https://github.com/Expensify/App/issues/45689